h2o-algos/src/main/java/hex/coxph/CoxPH.java

```diff
@@ -657,9 +658,16 @@ public void computeImpl() {
           calcCumhaz_0(model, coxMR);
         }
 
-        if (iterTimer != null)
+        if (iterTimer != null) {
           Log.info("CoxPH Last Iteration: " + iterTimer.toString());
 
+        }
+       
+        if (!_parms._skip_scoring) {
```

Comment:
it would be cleaner if the model was updated in DKV before scoring is invoked

Reply:
Added